docs/architecture/transaction-lifecycle.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Transaction lifecycle
+sidebar_position: 8
+---
+
+Transactions on Linea mainnet and testnet proceed through the following steps:
+
+## Step 1: Submission
+
+This is where every transaction originates: at the wallet level, a user signs a transaction and broadcasts it to Linea. From here, transactions head straight to the **mempool**, similarly to Ethereum, where they become pending transactions.
+
+## Step 2: Block building
+
+The Linea [sequencer](docs/architecture/sequencer) is responsible for order, building, and executing blocks. For each transaction added to the mempool, the sequencer checks its validity, rejecting transactions as necessary. Transaction validity conditions are specific to Linea, and differ slightly from those on other networks, including Ethereum.
+
+:::info
+
+A transaction is rejected if it:
+
+- Originates from a non-existent account, or an account that is not funded
+- Has a gas price below the minimum
+- Has a gas limit above the maximum
+- Has the same nonce as another transaction from the same account (in this case, the transaction with the higher gas fee is chosen, and the other rejected)
+- Has a total call data size greater than 25kb.
+
+:::
+
+The sequencer decides on the order in which transactions should be executed. So, having passed the above checks, valid transactions are placed into blocks in the correct sequence and executed.
+
+At this point, the transaction's lifecycle is more or less complete — at least from a user perspective. However, the transaction and its associated data will continue to be processed, in order to generate ZK proofs. Let's press on.
```

Comment:
As the block has been added to the head/tip of the Linea blockchain and a tx receipt is returned to the user. Let's link out to a page which describes the current block time. I think it's "Ethereum differences" currently

Reply:
Ah maybe because technically right now we have the same block time as Ethereum (12s). This will change very soon though, even with geth. Possibly down to 10 seconds or even 6 seconds. Plan for Alpha v2 is to go to 4s blocks. So we could just add a placeholder there for now with 12s and then update this in couple weeks with next minor release to, e..g 10 seconds

Reply:
okay, will leave for now!